src/libaktualizr/bootloader/bootloader.cc

```diff
@@ -95,6 +95,65 @@ void Bootloader::updateNotify() const {
   }
 }
 
+void Bootloader::installNotify(const Uptane::Target& target) const {
+  std::string sink;
+  std::string target_hash = target.sha256Hash();
+  std::string version_flag = "bootfirmware_version";
+  // TODO: provide initial /ostree/deploy/lmp/deploy/ from config
+  // instead of hardcoding that here
```

Comment:
Yeah, we should have a config entry for the distro/ostree branch and for the version.txt file path.